src/HeapDump/HeapDump.csproj

```diff
@@ -0,0 +1,83 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- net451 is actually just another net45 build build with the platform set to x64 -->
```

Comment:
This is a bit strange to use a net451 to represent the X64 build.     Is there not more straightforward way of doing this?  Can we simply make up a new targetFramework (e.g. net45-X64)?